content/authors/nag1201/_index.md

```diff
@@ -37,10 +37,10 @@ social:
  # link: <LINK TO YOUR TWITTER>
```

Comment:
maybe you should also comment line `36` `icon_pack: fab`?
also, leave no spaces before `#`.

Reply:
Yep, if you don't need the twitter block at all you can just remove it, so your social block would end up looking like
```
social:
  icon: github
  icon_pack: fab
  link: https://github.com/nag1201
```